test/mod/test_host.py

```diff
@@ -135,6 +147,41 @@ def test_exception():
             client.call("exception")
 
 
+def test_signals():
+    with host.ServiceManager() as mgr:
+        exec_callback = 0
+        def check_value(item, _fds):
```

Comment:
Super nitpick, autopep8 wants a newline here:

```suggestion
        exec_callback = 0
        
        def check_value(item, _fds):
```

Reply:
I think I'm gonna set autopep8 to run each time I save a file :smile: